packages/morph/package.json

```diff
@@ -10,16 +10,22 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@codemirror/lang-markdown": "^6.3.1",
+    "@codemirror/state": "^6.5.0",
+    "@codemirror/view": "^6.36.1",
     "class-variance-authority": "^0.7.1",
     "clsx": "^2.1.1",
+    "codemirror": "^6.0.1",
     "lucide-react": "^0.469.0",
+    "markdown-it": "^14.1.0",
```

Comment:
markdown-it's plugins ecosystem is not that nice to write imo.

if we want to expand upon the core commonmark then I don't think we should use it.